### PR TITLE
Issue #2208. Add edits to about page.

### PR DESCRIPTION
**Pull request by miketaylr**

### CODE REVIEW

File: webcompat/templates/about.html [hunk 1]
```diff
@@ -20,62 +20,62 @@ <h1 class="headline-1 highlight">About</h1>
...
         </p>
         <p>
           After moving the issue to diagnosis, one member of our community digs into the code to figure out what’s causing the compatibility issue and or files bugs with browser vendors.
         </p>
 
         <p>
-          Sometimes it might also be possible that the official specification is just partially done or there is an issue in there. We help to <strong class="bold">provide feedback</strong> and make suggestions to the specifications to make it <strong class="bold">stable for everyone</strong>.
+          Sometimes it might also be possible the relevant web technology standard is underspecified or is poorly implemented. We help to <strong class="bold">provide feedback</strong> and make suggestions to the specifications to make them <strong class="bold">stable for everyone</strong>.
```
Comment (zoepage) [inline webcompat/templates/about.html]: I'm not sure about the word `poorly`. I know, it makes sense for a native speaker and fits, but might sound not appropriate for a non-native speaker. How about `not well` or something similar?

Comment (miketaylr) [reply webcompat/templates/about.html]: Hmm. OK!

**Comment by miketaylr** [comment]
@zoepage changed that sentence to:

> Sometimes it might also be possible the relevant web technology standard is underspecified or is browsers implement it in different ways.

r? @zoepage

**Comment by miketaylr** [comment]
ah. welp. let me fix that typo haha.

**Pull request merged**